server/jetstream_cluster_test.go

```diff
@@ -10689,15 +10689,6 @@ func TestJetStreamClusterMirrorDeDupWindow(t *testing.T) {
 	// Send 100 messages
 	send(100)
 
-	// First check that we can't create with a duplicates window
```

Comment:
Why remove this test? This is testing that we can't create a mirror with de-dup, which caused some issues in the past.

Reply:
Which issues? I think its fine if you want to create a mirror with dupe window..